rclcpp/include/rclcpp/create_generic_subscription.hpp

```diff
@@ -54,7 +55,8 @@ std::shared_ptr<GenericSubscription> create_generic_subscription(
   std::function<void(std::shared_ptr<rclcpp::SerializedMessage>)> callback,
   const rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> & options = (
     rclcpp::SubscriptionOptionsWithAllocator<AllocatorT>()
-  )
+  ),
+  rclcpp::CallbackGroup::SharedPtr group = nullptr
```

Comment:
This variable should be `callback_group`, not just `group`, in my opinion.

Reply:
And everywhere else you added this argument.

Reply:
> storing the callback group in the subscription options as a weak pointer (maybe ok)

Yes, this is an alternative way.
To keep the same method with calling `create_wall_timer`, `create_client` and `create_service` by `callback_group`, I think to put the `callback_group` in the function argument could be acceptable as well.

> This variable should be callback_group, not just group, in my opinion.

OK. In addition to updating it imported by this PR, I'll update other variable names (e.g. [1](https://github.com/ros2/rclcpp/blob/16914e31a15417d5751c4cb611f591bbaa458eca/rclcpp/include/rclcpp/node.hpp#L241), [2](https://github.com/ros2/rclcpp/blob/16914e31a15417d5751c4cb611f591bbaa458eca/rclcpp/include/rclcpp/node.hpp#L255), etc.) that should be `callback_group`.

Reply:
Updated in https://github.com/ros2/rclcpp/pull/1833/commits/a9601ed095b99190a022d9683c37a7591501abf9